inmaputil/cmd.go

```diff
@@ -655,16 +666,23 @@ concentrations with no temporal variability.`,
 		if err != nil {
 			return err
 		}
+
+		shapeFiles := expandStringSlice(Cfg.GetStringSlice("EmissionsShapefiles"))
+		// wrap download over each filepaths
+		for _, element := range shapeFiles {
+			element = download(element, createChan(cmd))
```

Comment:
I'm not sure this would work as expected. You might need to do `shapeFiles[i] = download(...)`